README.md

```diff
@@ -99,6 +99,22 @@ Now download the files again:
 cargo run --bin safe --features local-discovery -- files download
 ```
 
+### Folders
+
+The folders storage capability can be demonstrated by uploading folders to the local network, then retrieving them.
+
+Upload a directory:
+```bash
+cargo run --bin safe --features local-discovery -- files upload <dir-path>
```

Comment:
is it shall be `folders` subcmd ?
or the `folders` subcmd is for download only ?